docs/tutorial.md

```diff
@@ -0,0 +1,159 @@
+      builder:
+        name: default-builder
+        kind: ClusterBuilder
+      source:
+        git:
+          url: <YOUR-GIT-URL>
+          revision: master
+    ```
+
+    - Make sure to replace <REGISTRY> with the registry you configured in step #2.     
+    - Make sure to replace <YOUR-GITHUB-URL> with a publicly accessible github url     
+
+   Apply that image to the cluster 
+    ```bash
+    kubectl apply -f <name-of-image-file.yaml>
+    ```
+    
+   You can now check the status of the image. 
+   
+   ```bash
+   kubectl get images 
+   ```
+    
+   You should see that the image has an unknown READY status as it currently building.
+   
+   ```
+    NAME                  LATESTIMAGE   READY
+    tutorial-image                      Unknown
+    ```
+    
+    You can tail the logs for image that is currently building by downloading the log utility for you operating system for the [github release](https://github.com/pivotal/kpack/releases)
+    
+    ```
+    logs -image tutorial-image  
+    ``` 
+    
+    Once the image finishes building you can get the fully resolved built image with `kubectl get`
+    
+    ```
+    kubectl get image tutorial-image
+    ```  
+    
+    The output should look something like this:
+    ```
+    NAMESPACE   NAME                  LATESTIMAGE                                                                                       READY
+    test        tutorial-image        gcr.io/project-name/app@sha256:6744b3b24a7ab8d2b45d7673313c180daccb534b3d931369d0aa9805712f34b8   True
+    ```
+    
+    The latest image is available to be used in locally via `docker pull` and in a kubernetes deployment.   
+
+1. kpack rebuilds with source code updates
+    
+   Push an update to the git repository configured in step #4. In a short amount of time, kpack should recognize the update and automatically rebuild your image.  
+    
+    You can see this happen by running `kubectl get builds`
+    ```
+    kubectl get builds
+    ``` 
+    You should see a new build with
+   
+   ```
+   NAME                                IMAGE                                                                                           SUCCEEDED
+   tutorial-image-build-1-8mqkc       gcr.io/project-name/app@sha256:6744b3b24a7ab8d2b45d7673313c180daccb534b3d931369d0aa9805712f34b   True
+   tutorial-image-build-2-xsf2l                                                                                                        Unknown
+   ```
+
+    You can tail the logs for the image with log utility used in step #5
+    
+    ```
+    logs -image tutorial-image -b 2  
+    ``` 
+1. kpack rebuilds with buildpack updates
+    
+    The next time the `cloudfoundry/cnb:bionic` is updated, kpack will detect if it contains buildpack updates to any of the buildpacks used by the tutorial image.
+    If there is a buildpack update, kpack will automatically create a new build to rebuild your image.    
+    
+### Next Steps
+
+- WIP
```

Comment:
We should "prove" that it worked here by have them do a `docker pull` and `docker run`